examples/advanced_circuits_algorithms/Randomness/Randomness_Generation.ipynb

```diff
@@ -918,9 +902,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.10"
+   "version": "3.7.12"
```

Comment:
Should this change ?

Reply:
All the above changes are due to notebook opened locally in IDE. I don't think these should matter much. We have other notebooks with similar changes: https://github.com/aws/amazon-braket-examples/blob/main/examples/braket_features/Verbatim_Compilation.ipynb